requirements.txt

```diff
@@ -15,6 +15,7 @@ isort
 pytest
 pytest-mock
 mypy
+types-pyyaml
```

Comment:
mypy is complaining about the yaml imports now in GitHub Actions:

```text
mypy --show-error-codes

uvicorn/config.py:21: error: Library stubs not installed for "yaml" (or incompatible with Python 3.8)  [import]
uvicorn/config.py:21: note: Hint: "python3 -m pip install types-PyYAML"
uvicorn/config.py:21: note: (or run "mypy --install-types" to install all missing stub packages)
uvicorn/config.py:21: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
tests/test_config.py:18: error: Library stubs not installed for "yaml" (or incompatible with Python 3.8)  [import]
```

Running `mypy --install-types` didn't seem to fix it (in GitHub Actions at least), so I installed the `types-pyyaml` package from the typeshed. Otherwise, we would have to add ignores, like `import yaml # type: ignore[import]`.

Reply:
Mypy 0.900 was released yesterday, and we don't pin the mypy version, jfyk.
https://mypy-lang.blogspot.com/2021/06/mypy-0900-released.html

Reply:
Can you try `mypy --install-types` instead of adding `types-pyyaml`?

Reply:
kind of happy this doesnt blow up more than that !